### ISSUE TITLE
TypeError: must be str, not bytes in ppt_parser.decompressor.decompress

### ISSUE THREAD

**Issue by jloehel** [decalage2/oletools#607]
https://github.com/decalage2/oletools/blob/d03c8683a81c1b4268b1cccb9333bbd7c0426f3f/oletools/ppt_parser.py#L1618

I guess this should be:
~~~
decomp = b''
~~~

Traceback:
```
Traceback (most recent call last):
...
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/olevba.py", line 2704, in __init__
    self.open_ppt()
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/olevba.py", line 3018, in open_ppt
    for vba_data in ppt.iter_vba_data():
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/ppt_parser.py", line 1147, in wrapped
    for result in func(self, self._open_main_stream, *args, **kwargs):
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/ppt_parser.py", line 1604, in iter_vba_data
    yield self.decompress_vba_storage(storage)
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/ppt_parser.py", line 1114, in wrapped
    return func(self, self._open_main_stream, *args, **kwargs)
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/ppt_parser.py", line 1555, in decompress_vba_storage
    iterative_decompress(stream, storage.data_size)
  File "/home/jloehel/projects/github.com/**/**/.venv/lib64/python3.6/site-packages/oletools/ppt_parser.py", line 1624, in iterative_decompress
    decomp += decompressor.decompress(stream.read(n_new))
TypeError: must be str, not bytes
```

**Comment by jloehel** [comment]
This is the same issue: #177 There is also already a PR open for this: #450. @decalage2 Any chance to get this merged soon? :-)

**Comment by decalage2** [comment]
I just merged PR #450 and now it works on the few PPT samples I tested. 
@jloehel could you please confirm it solves the issue on your side?

**Comment by jloehel** [comment]
> I just merged PR #450 and now it works on the few PPT samples I tested.
> @jloehel could you please confirm it solves the issue on your side?

Thanks for the quick response. It works fine now. :-) I can parse now the most of the samples. I experience more issues with PPT samples but I will report them in separated issues.
